RevenueCatUI/Purchasing/PurchaseHandler.swift

```diff
@@ -206,10 +208,12 @@ struct PurchasedResultPreferenceKey: PreferenceKey {
     struct PurchaseResult: Equatable {
         var transaction: StoreTransaction?
         var customerInfo: CustomerInfo
+        var userCancelled: Bool
```

Comment:
So in case this is true, the result would contain the preexisting customerInfo correct?